src/domains/checkout-app/api/checkout/checkout_carts_auth/v1/_base_policy.xml.tpl

```diff
@@ -11,6 +11,11 @@
                   <value>WISP_REDIRECT</value>
               </set-header>
           </when>
+          <when condition="@(context.Subscription != null && context.Subscription.Id.ToLower().Equals("ebollo"))">
```

Comment:
@infantesimone as agreed the `when` block has been replaced with `otherwise`, there's no need to distinguish between `ebollo` payments and any other ones coming from EC. 

cc @pietro-tota @pasqualespica